src/components/Feature/Feature.tsx

```diff
@@ -0,0 +1,114 @@
+/* eslint-disable valid-typeof */
+import Component from '../Component';
+import Subtitle from './Subtitle';
+import { Box, Container, Stack, Typography } from '@mui/material';
+
+export interface FeatureItem {
+  icon?: JSX.Element;
+  title: string;
+  subtitle: string;
+  linkTitle?: string;
+  url?: string;
+}
+
+export interface FeatureProps {
+  title: string;
+  items: FeatureItem[];
+  theme: 'dark' | 'light';
+}
+
+const Feature = ({ title, items, theme }: FeatureProps) => {
+  return (
+    <Component>
+      <Box bgcolor={theme === 'light' ? '#FAFAFA' : '#0062C3'}>
```

Comment:
@leleOFA for color it would be better to use theme colors (egs. primary...)

Reply:
do it